joomla-projects/gsoc21_media-manager
# Week 7: Responsive image size customization improvements

Pull Request for issue #19.

### Summary of Changes
- Add new form controls for plugin settings and content/form images to customize the image creation method and title.
- Fix the small issues related to responsive images functionality.


### Testing Instructions
Firstly, let's see how we can customize the default creation method (resizing) for all form and content images. Go to System -> Plugins -> Content - Responsive Images and choose the creation method you want:

![image](https://user-images.githubusercontent.com/62054743/127824884-037cad65-aa09-4f4a-b244-784e55cdaabd.png)

If you wish to set different configurations for content images, you can do it in the "Insert/Edit Image" form:

![image](https://user-images.githubusercontent.com/62054743/127825240-7780bd21-ff87-4e51-ac7b-31348e7a0e59.png)

And for form images:

![image](https://user-images.githubusercontent.com/62054743/127825444-5a903b66-f2cc-430b-80b2-77fdc6d6218f.png)
